CHANGELOG.md

```diff
@@ -2,14 +2,24 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.11.4] - 2024-07-31
+## [0.12.0] - 2024-08-29
+
+### Breaking change
+
+- 'from pasqal_cloud' import has completely replaced the deprecated import 'from sdk'
+- 'project' field has completely replaced the deprecated field 'group'
```

Comment:
field is `project_id` and this MR is simply removing `group_id`, We introduced project_id a while ago

Reply:
Should I write "'group' field is now removed, use 'project' field instead" ? Not sure how to make the message clear when we read the sentence.